pkg/aws/client.go

```diff
@@ -256,7 +256,7 @@ func (b *ClientBuilder) Build() (Client, error) {
 	}
 
 	if root {
-		return nil, errors.New("using a root account is not supported, please use an IAM user instead")
+		return nil, errors.New("Using a root account is not supported, please use an IAM user instead.")
```

Comment:
Now I'm getting
```
pkg/aws/client.go:259:26: error strings should not be capitalized or end with punctuation or a newline (golint)
		return nil, errors.New("Using a root account is not supported, please use an IAM user instead.")
		                       ^
make: *** [Makefile:45: lint] Error 1
```
I suspect that the linter fails after three issues per file, instead of giving you all the linting errors. You should try running `make lint` to see if these come out on your side. I don't recall seeing a linting error for this particular line until the last update :P